rakefile.netstd.rb

```diff
@@ -12,8 +12,9 @@
 solution        = "FakeItEasy.netstd.sln"
 assembly_info   = "src/CommonAssemblyInfo.cs"
 version         = IO.read(assembly_info)[/AssemblyInformationalVersion\("([^"]+)"\)/, 1]
-version_suffix  = ENV["VERSION_SUFFIX"]
-version_suffix  = version_suffix.to_s.empty? ? "-adhoc" : version_suffix
+#version_suffix  = ENV["VERSION_SUFFIX"]
+#version_suffix  = version_suffix.to_s.empty? ? "-adhoc" : version_suffix
+version_suffix  = "-netstd" #temporary while we have two build scripts

```

Comment:
This will interfere if ever we want to release a pre-release package, since the non-netstd package will get a -beta001 version, and this will have -netstd, losing the info about it being a beta. However, given that we'll likely have already released some -netstd package for the version we may be in trouble anyhow, since 

* -netstd will sort after -beta001-netstd (or similar) so the earlier package will be preferred, and
* -netstd will sort before -netstd-beta001, so once we finalize on a beta release, the earlier package would still be preferred.

Even after we merge #887, we have a similar problem with
* -netstd-build000334,
* -netstd-beta001-build000335, and
* -netstd-beta001-build000336 (final beta, not just some preliminary beta)

or 

* -netstd-build000334,
* -beta001-netstd-build000335, and
* -beta001-netstd-build000336 (final beta, not just some preliminary beta)

Of course, this may not come up at all. Perhaps the proposed numbering is fine. If ever we release a beta while still having a netstd package, we can just note that 2.4.0-netstd-build000789 is the "official .NET Standard beta001 for release 2.4.0". People consuming the AppVeyor feed probably don't care anyhow.
Okay! I think I've talked myself out of it being an issue. Regardless, it seems like changing to insert the netstd before or after the regular version number would not work. I'm content.